Makefile

```diff
@@ -22,6 +22,9 @@ check-fmt:
 test:
 	$(PYTHON3) -m pytest -p no:cacheprovider tests
 
+docker-run-image:
+	docker build -f docker/Dockerfile -t sdp-operator .
```

Comment:
Btw it would make sense to do the pip compile inside the docker image too.

Reply:
Updated to run pip-compile inside dev image